latest/docs-ref-conceptual/install-azure-cli.md

```diff
@@ -26,28 +26,28 @@ For information about the latest release, see the [release notes](release-notes-
 
 ## macOS
 
-> [!WARNING]
-> The Homebrew formula for the Azure CLI, `azure-cli`, is currently out of date and will install a previous version.
+1. Install `curl` via [homebrew](https://brew.sh/).
+
+   ```bash
+   brew install curl
```

Comment:
Verified that `curl` has been included since OS 10.9 ([Apple docsite link](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/curl.1.html)). Removed this step.